crates/bevy_sprite/src/texture_atlas.rs

```diff
@@ -128,15 +136,17 @@ impl TextureAtlas {
         self.textures.len() - 1
     }
 
-    /// How many textures are in the `TextureAtlas`
+    /// The number of textures in the [`TextureAtlas`]
     pub fn len(&self) -> usize {
         self.textures.len()
     }
 
+    /// Returns `true` if there are no textures in the [`TextureAtlas`]
     pub fn is_empty(&self) -> bool {
         self.textures.is_empty()
     }
 
+    /// Returns the index if texture in the [`TextureAtlas`]
```

Comment:
```suggestion
    /// Returns the index of the sub-image `Rect` corresponding to the given image handle in the [`TextureAtlas`],
    /// or `None` if there is no such sub-image.
```

Reply:
Given the context, I think we don't need to describe what a texture is, it is used in several places. e.g
```rust
    /// The number of textures in the [`TextureAtlas`]
    pub fn len(&self) -> usize {
        self.textures.len()
    }

    /// Returns `true` if there are no textures in the [`TextureAtlas`]
    pub fn is_empty(&self) -> bool {
        self.textures.is_empty()
    }
```

Reply:
how about: 
```
Returns the index of the texture corresponding to the given image handle in the [`TextureAtlas`]
```

Reply:
Yep looks good, keep it simple.